modules/wordads/php/class-wordads-ccpa-do-not-sell-link-widget.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
I may be missing something in that big diff, but shouldn't we require that new file in the main class now?

Reply:
I meant to do that, but ended up missing it. 183aa511b